test/util.mjs

```diff
@@ -0,0 +1,60 @@
+export const {
+  String,
+  RegExp,
+  Array: { isArray },
+  Reflect: { ownKeys, getOwnPropertyDescriptor },
+  JSON: { stringify: stringifyJSON },
+  undefined,
+  Object: {
+    hasOwn = (object, key) =>
+      getOwnPropertyDescriptor(object, key) !== undefined,
+  },
+} = globalThis;
+
+export const isSubsetJSON = (json1, json2, path) => {
```

Comment:
This is a bit of a misnomer: it's not comparing JSON at all, it's just doing structural object matching.